store/team.go

```diff
@@ -157,6 +161,16 @@ func (es *teamstore) GetTeams() []*Team {
 	return teams
 }
 
+// Used for test purpose
+func (es *teamstore) DeleteToken(token string) error {
+	es.m.Lock()
+	defer es.m.Unlock()
+
+	delete(es.tokens, token)
+
+	return nil
+}
+
```

Comment:
if we are not using as a functionality in haaukins, there is  no point to write it down for just test purpose.

Reply:
All right. I will delete it.